src/transformers/models/gpt_sw3/tokenization_gpt_sw3.py

```diff
@@ -9,10 +11,13 @@
     import torch
 
 from shutil import copyfile
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
 
 import sentencepiece as spm
 
+
+if TYPE_CHECKING:
+    from transformers.pipelines.conversational import Conversation
```

Comment:
nit: this should go after all of the main imports, and before `logger` definition e.g. [like here](https://github.com/huggingface/transformers/blob/9a5d468ba0562e2d5edf9da787881fa227132bca/src/transformers/models/blenderbot/tokenization_blenderbot_fast.py#L27)

Reply:
I've changed the order of the imports now. I also moved the `if is_torch_available(): ...` part down after the imports for consistency. Is that alright?